README.md

```diff
@@ -34,12 +34,15 @@ projects.
 
 ## Blog posts
 
-* 2016-12-01 ([Open Source](https://opensource.googleblog.com/2016/12/announcing-oss-fuzz-continuous-fuzzing.html),
+* 2016-12-01 - Announcing OSS-Fuzz: Continuous fuzzing for open source software
+([Open Source](https://opensource.googleblog.com/2016/12/announcing-oss-fuzz-continuous-fuzzing.html),
```

Comment:
let's use only one link here and in the next one, e.g. only security blog to be consistent across all three posts? I don't think three similar links give any extra value here

Reply:
Done!